client/tsconfig.json

```diff
@@ -14,8 +14,15 @@
     "incremental": true,
     "paths": {
       "@/*": ["./src/*"]
-    }
+    },
+    "target": "ES2020",
+    "typeRoots": ["./node_modules/@types", "./types"]
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    "src/pages/api/render-tikz.js"
```

Comment:
It looks like it should be "src/pages/render-tikz.**ts**" rather than .js. The idea behind adding something to `include` in tsconfig.json is so that the TypeScript compiler will compile it to js (so if you have a file declaring some types for example, you should add it here).

But, "**/*.ts" and "**/*.tsx" are already included -- which is a wildcard to include all ts and tsx files recursively in child directories. So you don't need to add "src/pages/api/render-tikz.ts" explicitly as it will already be captured by these wildcards.